### PR TITLE
Add website API documentation

### PR DESCRIPTION
**Pull request by necolas**

### CODE REVIEW

File: apps/website/docs/api/01-babel/01-preset.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+:::info[Web only]
+
+Configuring Babel to use this preset is required for web builds only. It should not be used for native builds.
+
+:::
+
+## Overview
+
+The preset ensures that production builds have **no runtime performance overhead on web** relative to using [React DOM](https://react.dev/) with an advanced atomic CSS solution like [StyleX](https://stylexjs.com). The preset is required to extract styles to static CSS and optimize element rendering.
```
Comment (nmn) [inline apps/website/docs/api/01-babel/01-preset.md:18]:
```suggestion
The preset ensures that production builds have **no additional runtime performance overhead on web** relative to using [React DOM](https://react.dev/) with an advanced atomic CSS solution like [StyleX](https://stylexjs.com). The preset is required to extract styles to static CSS and optimize element rendering.
```

File: apps/website/docs/api/02-css/01-create.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+const styles = css.create({
+  foo: {
+    backgroundColor: 'red'
+  },
+  bar: (color, padding) => ({
+    color: color,
+    padding: padding
+  })
+});
```
Comment (nmn) [inline apps/website/docs/api/02-css/01-create.md:26]:
```suggestion
const styles = css.create({
  foo: {
    backgroundColor: 'red'
  },
  bar: (color, padding) => ({
    color,
    padding,
  })
});
```

File: apps/website/docs/api/01-babel/01-preset.md (hunk 1 above)
Comment (necolas) [reply apps/website/docs/api/01-babel/01-preset.md:18]: "additional" is redundant given it says "relative to using..."

File: apps/website/docs/api/02-css/01-create.md (hunk 2 above)
Comment (necolas) [reply apps/website/docs/api/02-css/01-create.md:26]: This is intention for clarity in docs

File: apps/website/docs/api/01-babel/01-preset.md (hunk 1 above)
Comment (nmn) [reply apps/website/docs/api/01-babel/01-preset.md:18]: Yeah, I think the language can be a bit clearer even if the word "additional" *also* sounds a bit awkward.

**Review by nmn — APPROVED** [review]

**Pull request merged**